parser/src/main/java/parser/IBookParser.java

```diff
@@ -11,6 +11,7 @@
 /**
  * Created by grzegorz_sledz on 25.08.16.
  */
+
 public interface IBookParser {
 
     Logger logger = LogManager.getRootLogger();
```

Comment:
@Log4j2 annotation is legal on classes and enums. I can't use this annotation on interface. So it has done in traditional manner.